include/bluetooth/conn.h

```diff
@@ -31,6 +31,9 @@ extern "C" {
 /** Opaque type representing a connection to a remote device */
 struct bt_conn;
 
+/** Forward declaration of a IQ samples report struct */
+struct bt_df_conn_iq_samples_report;
```

Comment:
This is not really a forward declaration since the definition is not in this file. All public headers should be self-contained so you should add the missing #include statement instead

Reply:
I've removed this line and added `direction.h` include.